packages/polling-controller/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- StaticIntervalPollingController now allows to set duration to key ([#5683](https://github.com/MetaMask/core/pull/5683))
```

Comment:
The feature being added here wasn't initially clear to me based on this description. Should we be more specific? We should also mention the expect parts of the API that were added. I realize I made some comments below which means this is a bit in flux but at the very least perhaps we could reword this to:

``` suggestion
- StaticIntervalPollingController now allows starting a poll that will automatically stop after a given duration ([#5683](https://github.com/MetaMask/core/pull/5683))
```